espeakng/__init__.py

```diff
@@ -20,38 +20,39 @@
 import logging
 import subprocess
 import tempfile
+import platform
 
 class ESpeakNG(object):
 
-    def __init__(self, 
-                 volume      =         100, 
+    def __init__(self,
+                 volume      =         100,
                  audio_dev   =        None,
                  word_gap    =          -1, # ms
                  capitals    =           0, # indicate capital letters with: 1=sound, 2=the word "capitals", higher values indicate a pitch increase (try -k20).
                  line_length =           0, # Line length. If not zero, consider lines less than this length as end-of-clause
                  pitch       =          50, # 0-99
-                 speed       =         175, # approx. words per minute 
-                 voice       = 'english-us'):
+                 speed       =         175, # approx. words per minute
+                 voice       = 'en'):
 
 
-        self._volume      = volume  
-        self._audio_dev   = audio_dev  
-        self._word_gap    = word_gap   
-        self._capitals    = capitals   
+        self._volume      = volume
+        self._audio_dev   = audio_dev
+        self._word_gap    = word_gap
+        self._capitals    = capitals
         self._line_length = line_length
-        self._pitch       = pitch      
-        self._speed       = speed      
-        self._voice       = voice      
+        self._pitch       = pitch
+        self._speed       = speed
+        self._voice       = voice
 
     def _espeak_exe(self, args, sync=False):
-        cmd = ['espeak-ng', 
+        cmd = ['espeak' if platform.system()=='Darwin' else 'espeak-ng',
```

Comment:
I searched in https://brew.sh/ and found espeak, but no espeak-ng, I guess you can only brew install espeak
I hope this will solve #1 , *but I don't have a mac to confirm* . I tested it on Ubuntu18 and Win10
I have a little robot project, it's tts function depends on py-espeak-ng. I hope it support all platforms

Reply:
This looks like a hack to me. We could look into auto-detecting the availability of first espeak-ng and, as a fallback espeak on a system and use the best available option.